gradoop-temporal/src/test/resources/log4j-test.properties

```diff
@@ -0,0 +1,11 @@
+# Set root logger level to OFF to not flood build logs
```

Comment:
Not really part of this PR.

Reply:
I agree, looks like this was accidentally added